atomics/T1218/T1218.yaml

```diff
@@ -223,3 +223,30 @@ atomic_tests:
   executor:
     command: 'Invoke-ATHRemoteFXvGPUDisablementCommand -ModuleName #{module_name} -ModulePath #{module_path}'
     name: powershell
+- name: DiskShadow Command Execution
+  description: |
+    Emulates attack with a DiskShadow.exe being used to execute arbitrary commands
```

Comment:
Could you include the link to the blog post that you have in the Pull Request description, it is excellent and a good reference for everyone running this atomic. Thx

Reply:
Thanks for the feedback, I've expanded the description to include a reference with the link from the PR.